build/gulp/tasks/stats.ts

```diff
@@ -0,0 +1,126 @@
+import * as fs from 'fs'
+import { task, parallel, series } from 'gulp'
+import * as _ from 'lodash'
+import * as webpack from 'webpack'
+import * as stableStringify from 'json-stable-stringify-without-jsonify'
+
+import config from '../../../config'
+
+const g = require('gulp-load-plugins')()
+const { paths } = config
+const { log, PluginError } = g.util
+
+const UNRELEASED_VERSION_STRING = 'Unreleased'
+const SEMVER_MATCHER = /(\d+)\.(\d+)\.(\d+)/
+
+const semverCmp = (a, b) => {
```

Comment:
Looks quite confusing that we need to have a custom function to compare semver 🤔

Reply:
Yes, we can use a package for that, but then we need to handle `UNRELEASED_VERSION_STRING` explicitly anyway.
Moreover, this will probably be removed once we implement external storage for bundle size.

Reply:
even given this requirement, it seems that the task would reduce to
- filter out versions with UNRELEASED_VERSION_STRING
  - put them first
- compare rest entries using logic of dedicated package

This approach would be much more preferable to my mind.